packages/fxa-payments-server/src/routes/Subscriptions/PaymentUpdateForm.tsx

```diff
@@ -102,17 +155,17 @@ export const PaymentUpdateForm = ({
   }, [paymentError, setPaymentError]);
 
   const onFormMounted = useCallback(() => Amplitude.updatePaymentMounted(), []);
-
   const onFormEngaged = useCallback(() => Amplitude.updatePaymentEngaged(), []);
 
-  const { last4, exp_month, exp_year, payment_provider } = customer;
-
   // https://github.com/iamkun/dayjs/issues/639
   const expirationDate = dayjs()
     .set('month', Number(exp_month) - 1)
     .set('year', Number(exp_year))
     .format('MMMM YYYY');
 
+  const currencyCode = plan && plan.currency ? plan.currency : 'UNKNOWN';
+  const planId = plan && plan.plan_id ? plan.plan_id : 'UNKNOWN';
+
```

Comment:
@biancadanforth already verified that the customer won't see the subscriptions management page without the plans, so we are now discussing handling potential coding errors... if an error lead to a missing plan or `plan_id` or `currency`, we should show an error.  Setting these defaults will only delay an unrecoverable error for the user.  We already know there's an error at this point; let's handle it here.